vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -0,0 +1,143 @@
+package org.semanticweb.vlog4j.core.model.implementation;
+
+import java.util.List;
```

Comment:
remove unused imports. You can see them as warnings, in yellow in Eclipse. Please fix all warnings.